Rick-and-Morty/Rick And Morty/Services/RickAndMortyService.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  RickAndMortyService.swift
+//  Rick And Morty
+//
+//  Created by Scottie Gray on 2021-08-10.
+//  Copyright © 2021 Novoda. All rights reserved.
+//
+
+import Foundation
+
+final class RickAndMortyService {
```

Comment:
Why not make this conform to a `RickAndMortyServiceProtocol` that contains `func fetchData(url: URL, success: @escaping (Dictionary<String, Any>) -> (), error: @escaping (Error?) -> ())` that way you are better prepared for dependency injection and testing :)